### PR TITLE
improving user culling docs

### PR DESCRIPTION
**Pull request by choldgraf**
Documentation on user culling behavior.

### CODE REVIEW

File: doc/source/user-management.md [hunk 1]
```diff
@@ -3,6 +3,51 @@
 This section describes management of users and their
 permissions on JupyterHub.
 
+## Culling user pods
+JupyterHub will automatically delete any user pods that have no activity
+for a period of time. This helps free up computational resources and keeps
+costs down if you are using an autoscaling cluster.
+When these users navigate back to your JupyterHub, they will
+have to start their server again, and the state of their previous session
```
Comment (yuvipanda) [inline doc/source/user-management.md]: Should we explain what is 'state' here?

File: doc/source/user-management.md [hunk 2]
```diff
@@ -3,6 +3,51 @@
...
+will be lost. This is known as "culling".
```
Comment (yuvipanda) [inline doc/source/user-management.md]: culling should be bold or italics?

File: doc/source/user-management.md [hunk 3]
```diff
@@ -3,6 +3,51 @@
...
+
+```eval_rst
+
+.. note::
```
Comment (yuvipanda) [inline doc/source/user-management.md]: Is there a reason this is a ..note rather than just a paragraph?

Comment (choldgraf) [reply doc/source/user-management.md]: to me the paragraph about inactivity is *related* to culling but kind of breaks the flow of the culling configuration instructions. It feels like an aside, which is why I made it a note.

**Comment by choldgraf** [comment]
comments addressed!

**Pull request merged**